newrelic/core/agent_protocol.py

```diff
@@ -50,6 +50,131 @@
 _logger = logging.getLogger(__name__)
 
 
+# {return_value: ...} wrapper has been removed from each of these
+FAKE_AGENT_RESPONSE = {
+    'preconnect': {"redirect_host": "collector.newrelic.com"},
+    'connect': json_decode(r"""
```

Comment:
We may want to move the `connect` response to a Django setting so we can iterate on it if needed. But I suspect it's fine as is while we do the experiment.